### ISSUE TITLE
[Bug] Special chars in database options

### ISSUE THREAD

**Issue by SergeyAB** [exodus4d/pathfinder#452]
If DB_USER consist a special chars like "-" (for example: `pathfinder-my-alliance-short-name`) setup will crash after pressing "Setup tables" button with error: 

> Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '-my-alliance-short-name CHARACTER SET utf8 COLLATE utf8_general_ci' at line 1

**Comment by MorpheusXAUT** [comment]
Just tried reproducing this with a throwaway DigitalOcean LEMP instance, but couldn't find any issue with hyphens in username, password or DB name.

Do you by any chance have a `,` in any of your DB settings? Apparently FatFreeFramework parses these config values differently. Additionally, you could try enclosing the config values that cause issues in double quotes `"`, which might help with possible config parsing issues according to the [SMTP wiki page](https://github.com/exodus4d/pathfinder/wiki/SMTP).

An example screenshot with a similar name to the one you mentioned (couldn't use the full one since it was longer than 32 chars):
![GitHub Logo](http://i.imgur.com/5WSdtV7.png)

**Comment by SergeyAB** [comment]
DB Status `connected` but error will shows only after pressing "Setup tables" (for my clean install).
Double Quotes didn't help me and I renamed my DB User for single world.

**Comment by MorpheusXAUT** [comment]
Strangely enough, I didn't have any errors displayed after pressing "setup tables" and "fix keys", pathfinder apparently was satisfied with the tables created, as indicated by the checkmarks next to each table.

I'll try to poke at it a bit more to see if I can reproduce it. What software versions (pathfinder, webserver, database) were you using?

**Comment by SergeyAB** [comment]
DB Status `connected` but error will shows only after pressing "Setup tables" (for my clean install).
Double Quotes didn't help me and I renamed my DB User for single world.

Sorry, my environment:
> $ uname -a
> FreeBSD gwtools.ru 11.0-RELEASE-p8 FreeBSD 11.0-RELEASE-p8 #0

> $ php -v
> PHP 5.6.30 (cli) (built: Feb 25 2017 01:27:00)
> Copyright (c) 1997-2016 The PHP Group
> Zend Engine v2.6.0, Copyright (c) 1998-2016 Zend Technologies

> $ mysql -V
> mysql  Ver 14.14 Distrib 5.6.35, for FreeBSD11.0 (amd64) using  EditLine wrapper

PHP 5.6, yeah but I use pathfinder v. 1.1.7 because I can't update PHP to 7.* version atm.

**Comment by MorpheusXAUT** [comment]
@SergeyAB I was able to reproduce your issue, although it was being caused by a database **name** having a hyphen, not actually a user.

This snippet shows the code cause the error for your install, executed in the MySQL shell manually:
```
mysql> ALTER DATABASE pathfinder-test CHARACTER SET utf8 COLLATE utf8_general_ci;
ERROR 1064 (42000): You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '-test CHARACTER SET utf8 COLLATE utf8_general_ci' at line 1
```
Since the `pathfinder-test` database name is not properly enclosed in quotes, MySQL interprets the command incorrectly and thus throws an error. The following SQL command works:
```
mysql> ALTER DATABASE `pathfinder-test` CHARACTER SET utf8 COLLATE utf8_general_ci;
Query OK, 1 row affected (0.00 sec)
```
Note the additional ``` ` ``` surrounding the database name, thus properly escaping it for MySQL.

From my digging so far, I've found two problemativ queries [here](https://github.com/exodus4d/pathfinder/blob/master/app/main/controller/setup.php#L928) and [here](https://github.com/exodus4d/pathfinder/blob/master/app/main/db/sql/mysql/tablemodifier.php#L55). Both places are missing the required ``` ` ``` quotes. I'll discuss this with @exodus4d on Slack real quick and send a PR to fix this issue.
